com.doomretro.DOOM-Retro.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: com.doomretro.DOOM-Retro
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "23.08"
+command: doomretro
+
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+
+  # We redirect the original ~/.config/doomretro to work as a configuration folder and WAD storage folder
+  - --env=DOOMWADDIR=.config/doomretro
+  - --env=SDL_SOUNDFONTS=/app/share/sounds/sf2/gzdoom.sf2  # FIXME: This doesn't appear to work.
+  - --persist=.config/doomretro
+
+  # Allow controller support
+  - --device=all
+
+modules:
+
+  - name: doomretro
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+
+    sources:
+      - type: git
+        url: https://github.com/bradharding/doomretro.git
+        tag: v5.1
+        commit: e991824829c97ed09cda799b74984a95cc0498c0
+        x-checker-data:
+          type: anitya
+          project-id: 17531
+          stable-only: true
+          tag-template: v$version
+      - type: file
+        url: https://github.com/zdoom/gzdoom/raw/g4.11.3/soundfont/gzdoom.sf2
+        sha256: a9058609b07c1f0e0c1f420e76c2e80ba78136e6173e836caab51ffb13e552f1
+
+    post-install:
+      - install -Dm 644 gzdoom.sf2 /app/share/sounds/sf2/gzdoom.sf2
+
+    cleanup:
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/pkgconfig
+
+  - shared-modules/linux-audio/fluidsynth2.json
```

Comment:
this is build AFTER. that's probaby why it doesn't work.

Reply:
I've tried moving this module on top of doomretro but it didn't resolve the issue.

doomretro itself doesn't say a lot about the error:

![image](https://github.com/flathub/flathub/assets/180032/acd83a90-9260-481c-96cc-e310fc7c85d0)